### PR TITLE
Update 1password logo

### PR DESCRIPTION
**Pull request by berkbaski**
1password logo was broken and then i just fixed this.

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Looks fine for me as-is? Where do you see it being broken?

**Comment by berkbaski** [comment]
Hey! I seen on README.md file.

Screen Shot 2021-04-05 at 16 43 07

**Comment by EricFromCanada** [comment]
The [cached version](https://camo.githubusercontent.com/e0a327e5f4022c6f89d2646589cb6a3528ca38601910ac42d545d8b4cb7cd448/68747470733a2f2f3170617373776f72642e636f6d2f696d672f70726573732f6c6f676f2e65356266303862303831353765303861316239313132666364353861353462362e737667) still loads, but the [original](https://1password.com/img/press/logo.e5bf08b08157e08a1b9112fcd58a54b6.svg) has been removed.

File: README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Our bottles (binary packages) are hosted by [Bintray](https://bintray.com/homebr
 
 Secure password storage and syncing is provided by [1Password for Teams](https://1password.com/teams/).
 
-[![1Password](https://1password.com/img/press/logo.e5bf08b08157e08a1b9112fcd58a54b6.svg)](https://1password.com)
+[![1Password](https://1password.com/img/redesign/logo-light-bg.svg)](https://1password.com)
```
Comment (EricFromCanada) [inline README.md:95]:
```suggestion
[![1Password](https://1password.com/img/redesign/press/logo.c757be5591a513da9c768f8b80829318.svg)](https://1password.com)
```

**Review by MikeMcQuaid — APPROVED** [review]
Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @berkbaski!

**Pull request merged**